entrypoint.sh

```diff
@@ -59,7 +65,7 @@ function trigger_workflow {
     -H "Accept: application/vnd.github.everest-preview+json" \
     -H "Content-Type: application/json" \
     -H "Authorization: Bearer ${INPUT_GITHUB_TOKEN}" \
-    --data "{\"event_type\": \"${event_type}\", \"client_payload\": {} }"
+    --data "{\"event_type\": \"${event_type}\", \"client_payload\": ${client_payload} }"
```

Comment:
Do you  need \" around variable like ` \"${client_payload}\" ` ?

Reply:
I can certainly add them and we can test.

Reply:
I just tested this and it doesn't work with \".  It does work without it.  When we next have a slack call ask me abou this.

Reply:
Together we determined that we can expand the variable in the payload if we use

```
${{ github.repository }}
```

Thanks for your help.